classes/01class/exercises/c01-git01/afaryy/README.md

```diff
@@ -0,0 +1,57 @@
+# c01-git01
+
+## Commands executed
+
+1. Commands typed to solve the "Merging" exercise
+```
+--Make a new branch called bugFix and Checkout the bugFix branch
+git checkout -b bugFix
+
+--Commit once
+git commit
+
+--Go back to master with git checkout
+git checkout master
+
+--Commit another time
+git commit
+
+--Merge the branch bugFix into master with git merge
+git merge bugFix
+
+--Merge the branch master into bugFix with git merge
+git checkout bugFix; git merge master
+
```

Comment:
I dont think this is needed for that exercise.

It makes sense if you are continuing the development in "bugFix" to merge master into it (or rebase) -- but for this exercise this is extra (merging master back into bugFix)